src/clims/runner/commands/create_example_data.py

```diff
@@ -11,10 +12,32 @@ def createexampledata():
     """
     Create example data for clims
     """
+    from clims.services.application import ioc, ApplicationService
+    from clims.services.substance import SubstanceBase, FloatField
+    from clims.models.plugin_registration import PluginRegistration
+    from sentry.models.organization import Organization
 
-    # Import models here because they need to be wrapped with a Django context to work.
-    # /JD 2019-05-27
-    from clims.models.work_batch import WorkBatch
-    WorkBatch.objects.create(name="Test1", organization_id=1, handler="somehandler")
-    WorkBatch.objects.create(name="Test2", organization_id=1, handler="somehandler2")
-    WorkBatch.objects.create(name="Test3", organization_id=1, handler="somehandler2")
+    app = ApplicationService()
+    ioc.set_application(app)
```

Comment:
Note to self: We should allow that `ioc` to return a default version of the application if no one has called `set_application`. Then this kind of setup would be required in e.g. this case.